client/index.jsx

```diff
@@ -41,12 +44,49 @@ class Pictures extends React.Component {
       });
   }
 
+  handleButtonClick(e) {
```

Comment:
Same comment about spelling things out - this one is less important because many, many people use e but I prefer event because explicit is better AND (at least for me) it helps remind me this is based on an action that happens somewhere else. If I stop and think about what e means I'll remember but when I read event it brings that extra information with it. 100% just my opinion.